rudder-web/src/main/webapp/javascript/rudder/homePage.js

```diff
@@ -234,10 +234,17 @@ function doughnutChart (id,data,count,colors) {
                         , value     : data[0]._model.label
                        }];
                       break;
+                  case 'nodeCompliance':
+                     var compliance = {
+                        "Poor"   : "[0,50]"
+                       ,"Average": "[50,75]"
+                       ,"Good"   : "[75,100]"
+                       ,"Perfect": "[100,100]"
+                       };
```

Comment:
Here you could create an object "var complianceFilter = { complianceFilter : compliance[data[0]._model.label.split(' ')[0]] }

Reply:
In fact you should also have a compliance[data[0]._model.label.split(' ')[0]] stored in a var (let's call it complianceLevel )